src/data/Register.jsx

```diff
@@ -0,0 +1,9 @@
+export const requirphotography = {
```

Comment:
create a single requirement array and have an array of objects, reference https://github.com/acm-ucr/hackathon-website/blob/dev/src/data/Volunteers.js#L5

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/122486013/223096f6-b260-494d-97a5-b594ca8f3f86)
like this?